src/main/kotlin/com/exactpro/th2/codec/util/ErrorMessageUtil.kt

```diff
@@ -44,38 +42,3 @@ fun RawMessage.toErrorMessage(protocols: Collection<String>, errorEventId: Event
     it[ERROR_EVENT_ID] = errorEventId
 }
 
-fun MessageGroup.toErrorGroup(infoMessage: String,
```

Comment:
Is it necessary to replace it? I think it is better to store it like util fiel for future usage instead of decode method